corehq/apps/app_manager/tests/data/suite/suite-case-detail-instances.xml

```diff
@@ -155,7 +155,7 @@
           <locale id="m1.case_short.case_name_1.header"/>
         </text>
       </header>
-      <template>
+      <template form="plain">
```

Comment:
I'm generally nervous about changes to this file - `plain` is used by HQ but previously wasn't passed to mobile and it isn't one of the [expected forms](https://github.com/dimagi/commcare-hq/blob/27ec9170cf83c779e8bccf2db5df8ee38ae0f4c4/corehq/apps/app_manager/suite_xml/xml_models.py#L640). I don't _think_ those forms are validated, but can you validate that this change won't break mobile?

Reply:
...for some reason I thought this was the default template file, but it's just a test file. Nevermind.